numbat/src/value.rs

```diff
@@ -28,6 +28,7 @@ pub enum Value {
     /// A DateTime with an associated offset used when pretty printing
     DateTime(chrono::DateTime<chrono::Utc>, chrono::FixedOffset),
     FunctionReference(FunctionReference),
+    FormatSpecifier(String),
```

Comment:
Hm, this feels wrong(?). A format specifier can't live as a standalone value, right? You probably added it because you needed to implement `Constant::to_value`? We might need to rethink that.

Reply:
> Hm, this feels wrong(?). A format specifier can't live as a standalone value, right?

Agreed.

> You probably added it because you needed to implement `Constant::to_value`? We might need to rethink that.

Indeed.

Reply:
Let's maybe revert the changes in value.rs for now and implement `to_value` using `unreachable!(…)`? See if that works out?

Reply:
Oh right. Let's leave things as they are and clean this up later. Maybe a constant can be a `Value`. Or a `Constant(Value)` or similar.